src/commands/generate.js

```diff
@@ -91,7 +91,7 @@ module.exports = function () {
       var lastLineIndex = indexLines.length - 1;
       var newLine = `export { default as ${name} } from "./${name}"`;
       if (indexLines[lastLineIndex] === "") {
-        indexLines.splice(lastLineIndex - 1, 1, newLine);
+        indexLines.splice(lastLineIndex, 1, newLine);
```

Comment:
Adding a note here on how to replicate. I ran into this when testing out the destroy commands. [Array.splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) will start deleting at the index it is supplied so by passing `lastLineIndex - 1` you could potentially move to a line with a reducer and delete it by accident.

Using gluestick before this commit
```
gluestick new testapp
gluestick generate reducer testone
```
Now `src/reducers/index.js` should look like this
<img width="835" alt="screen shot 2016-01-25 at 11 36 36 pm" src="https://cloud.githubusercontent.com/assets/1280586/12574863/8c8f02b6-c3bc-11e5-9092-41c33c3edd3d.png">

But sometimes I have weird tendencies and decide to delete white space and so I changed `src/reducers/index.js` to look like this:
<img width="718" alt="screen shot 2016-01-25 at 11 37 24 pm" src="https://cloud.githubusercontent.com/assets/1280586/12574874/9f727354-c3bc-11e5-9abe-8fb2f147deb9.png">

Then I ran
`gluestick generate reducer testtwo` and `src/reducers/index.js` looked like this:
<img width="768" alt="screen shot 2016-01-25 at 11 38 00 pm" src="https://cloud.githubusercontent.com/assets/1280586/12574881/b4f5265e-c3bc-11e5-999f-7f0622efae73.png">

I'm not sure if you had specific intentions here, but I don't think having reducers overwrite each other because someone has made modifications to `src/reducers/index.js` is a desired effect.